docs/rfc/README.md

```diff
@@ -53,4 +53,7 @@ sections.
 - [RFC-013: ABCI++](./rfc-013-abci++.md)
 - [RFC-014: Semantic Versioning](./rfc-014-semantic-versioning.md)
 
+
```

Comment:
```suggestion
```

Reply:
This is a placeholder for ongoing RFC016 and RFC017